form-los-altos-urgent-report.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-type" content="text/html;charset=UTF-8">
+    <link rel="stylesheet" type="text/css" href="resources/css/loading.css"/>
+    <link rel="stylesheet" type="text/css" href="resources/css/pack-it-forms.css"/>
+    <style>
+      table {
+          width: 100%;
+      }
+      table.noborder {
+          border: 0px;
+          margin-left: 0px;
+      }
+      td.label {
+          font-weight: bold;
+          text-align: right;
+          vertical-align: middle;
+      }
+    </style>
+    <script type="text/javascript" src="resources/js/pack-it-forms.js"></script>
+    <title>Urgent Report</title>
+  </head>
+
+  <body>
+    <div class="version">0.1.1</div>
+    <div id="loading"><div>Loading<div id="spin"><div id="spin_1" class="spin"></div></div></div></div>
+    <div id="err">
+      <div>
+        <h2>Something went wrong.</h2>
+        This information might help resolve the problem:
+        <div id="error-log"></div>
+      </div>
+    </div>
+    <form id="the-form" name="the-form" style="background-color:#ffffff;">
+      <h1>Urgent Report</h1>
+      <table>
+        <tr>
+          <td class="label">Location:</td>
+          <td style="width:60%;"><input name="10.subject" type="text" required/></td>
+          <td class="label">Our Msg #</td>
+          <td>
+            <input name="MsgNo" type="text" class="no-msg-init" required="required"
+                   value="{{expand-while-null:{{envelope:RCVNUM}},{{query-string:msgno}}}}"/>
+          </td>
+        </tr><tr>
+          <td class="label">Cross Street:</td>
+          <td><input name="22.crossStreet" type="text"/></td>
+          <td class="label">Date:</td>
+          <td>
+            <input name="1a.date" class="" type="text" size="12"
+                   required="required" pattern="\d{1,2}/\d{1,2}/\d{4}"
+                   value="{{date}}"/>
+          </td>
+        </tr><tr>
+          <td rowspan="4" class="label" style="vertical-align:top">Description:</td>
+          <td rowspan="4">
+            <textarea name="23.description" rows="5" required="required"></textarea>
+          </td>
+          <td class="label">Time:</td>
+          <td>
+            <input name="1b.time" class="" type="text" size="10"
+                   required="required" pattern="\d{1,2}:\d{2}(:\d{2})?"
+                   value="{{time}}"/>
+          </td>
+        </tr><tr>
+          <td class="label">BAT #</td>
+          <td><input name="20.BAT" type="text" required/></td>
+        </tr><tr>
+          <td class="label">Reported by:</td>
+          <td><input name="21.reporter" type="text"/></td>
+        </tr><tr>
```

Comment:
Is "Reported by" always expected to be the radio operator, or could this be a BAT leader or some other non-ham?
Maybe we should have separate fields for the radio operator and the reporter? Whatever we choose to do, I'd like to make sure that the operator call (whether a personal one or a tactical one) is included in the message and that we default it to the operator callsign that Outpost provides.

Reply:
That's a question for Jim Clark, I think. I forwarded it to him via email.